fem/gridfunc.hpp

```diff
@@ -345,14 +345,32 @@ class GridFunction : public Vector
        projection matrix. */
    void ProjectGridFunction(const GridFunction &src);
 
+   /** @brief Project @a coeff Coefficient element-wise to @a this GridFunction.
+       The projection computation used on each element depends on the choice of
+       FiniteElementSpace #fes. Note that this is not generally an orthogonal
+       projection but may be an element-wise interpolation at element
+       integration points. */
```

Comment:
I wonder if the term "integration points" will be misleading.  It seems that you're trying to make the point that this is not computing integrals of the coefficient.  Perhaps something more vague like "at select points within each element".